public/css/text_arrange.css

```diff
@@ -0,0 +1,6 @@
+.ftable {
```

Comment:
If you are using bootstrap you can just use `d-flex` class. Read here: https://getbootstrap.com/docs/4.0/utilities/flex/